src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -47,12 +47,22 @@ namespace Raven.Server.Documents.Handlers
 {
     public class DocumentHandler : DatabaseRequestHandler
     {
+
+        public DocumentHandler()
+        {
+            if (TrafficWatchManager.HasRegisteredClients)
+                AddStringToHttpContext("N/A", TrafficWatchChangeType.Documents);
+        }
+
         [RavenAction("/databases/*/docs", "HEAD", AuthorizationStatus.ValidUser, EndpointType.Read)]
         public Task Head()
         {
             var id = GetQueryStringValueAndAssertIfSingleAndNotEmpty("id");
             var changeVector = GetStringFromHeaders(Constants.Headers.IfNoneMatch);
 
+            if (TrafficWatchManager.HasRegisteredClients)
```

Comment:
we can remove this since we log in the ctor now

Reply:
Done